test/sanity/pslint/ignore.txt

```diff
@@ -1,180 +1,106 @@
 test/integration/targets/win_csharp_utils/library/ansible_basic_tests.ps1 PSCustomUseLiteralPath
 test/integration/targets/win_csharp_utils/library/ansible_basic_tests.ps1 PSUseDeclaredVarsMoreThanAssignments # test setup requires vars to be set globally and not referenced in the same scope
 test/integration/targets/win_csharp_utils/library/ansible_become_tests.ps1 PSCustomUseLiteralPath
 test/integration/targets/win_exec_wrapper/library/test_fail.ps1 PSCustomUseLiteralPath
 test/integration/targets/win_iis_webbinding/library/test_get_webbindings.ps1 PSUseApprovedVerbs
-test/integration/targets/win_module_utils/library/argv_parser_test.ps1 PSAvoidTrailingWhitespace
 test/integration/targets/win_module_utils/library/argv_parser_test.ps1 PSUseApprovedVerbs
 test/integration/targets/win_module_utils/library/backup_file_test.ps1 PSCustomUseLiteralPath
-test/integration/targets/win_module_utils/library/camel_conversion_test.ps1 PSUseDeclaredVarsMoreThanAssignments
 test/integration/targets/win_module_utils/library/command_util_test.ps1 PSCustomUseLiteralPath
 test/integration/targets/win_psmodule/files/setup_modules.ps1 PSCustomUseLiteralPath
-test/integration/targets/win_ping/library/win_ping_strict_mode_error.ps1 PSUseDeclaredVarsMoreThanAssignments
 test/integration/targets/win_reboot/templates/post_reboot.ps1 PSCustomUseLiteralPath
-test/integration/targets/win_script/files/test_script.ps1 PSAvoidUsingWriteHost
 test/integration/targets/win_script/files/test_script_creates_file.ps1 PSAvoidUsingCmdletAliases
 test/integration/targets/win_script/files/test_script_removes_file.ps1 PSCustomUseLiteralPath
-test/integration/targets/win_script/files/test_script_with_args.ps1 PSAvoidUsingWriteHost
-test/integration/targets/win_script/files/test_script_with_splatting.ps1 PSAvoidUsingWriteHost
 test/integration/targets/win_stat/library/test_symlink_file.ps1 PSCustomUseLiteralPath
 test/integration/targets/win_user_right/library/test_get_right.ps1 PSCustomUseLiteralPath
 test/runner/setup/windows-httptester.ps1 PSCustomUseLiteralPath
+test/integration/targets/win_script/files/test_script.ps1 PSAvoidUsingWriteHost # Keep
+test/integration/targets/win_script/files/test_script_with_args.ps1 PSAvoidUsingWriteHost # Keep
+test/integration/targets/win_script/files/test_script_with_splatting.ps1 PSAvoidUsingWriteHost # Keep
+lib/ansible/modules/windows/win_domain.ps1 PSAvoidUsingEmptyCatchBlock # Keep
+lib/ansible/modules/windows/win_dsc.ps1 PSAvoidUsingEmptyCatchBlock # Keep
+lib/ansible/modules/windows/win_find.ps1 PSAvoidUsingEmptyCatchBlock # Keep
+lib/ansible/modules/windows/win_region.ps1 PSAvoidUsingEmptyCatchBlock # Keep
+lib/ansible/modules/windows/win_uri.ps1 PSAvoidUsingEmptyCatchBlock # Keep
+lib/ansible/modules/windows/win_domain.ps1 PSUseDeclaredVarsMoreThanAssignments
+lib/ansible/modules/windows/win_domain_controller.ps1 PSUseDeclaredVarsMoreThanAssignments
```

Comment:
This would be much easier to review if these entries remained on their original lines, since we want to make sure new entries are not being added.

Reply:
I moved them as to the end to have a better order in the file for future work